zerver/webhooks/buildbot/view.py

```diff
@@ -0,0 +1,33 @@
+from typing import Dict, Any
+from datetime import datetime
+
+from django.http import HttpRequest, HttpResponse
+
+from zerver.decorator import api_key_only_webhook_view
+from zerver.lib.webhooks.common import check_send_webhook_message
+from zerver.lib.request import REQ, has_request_variables
+from zerver.models import UserProfile
+from zerver.lib.response import json_success
+
+@api_key_only_webhook_view('Buildbot')
+@has_request_variables
+def api_buildbot_webhook(request: HttpRequest, user_profile: UserProfile,
+                         payload: Dict[str, Any]=REQ(argument_type='body')) -> HttpResponse:
+    topic = payload["project"]
+    if topic == "":
```

Comment:
Not a huge deal but I'd recommend just doing `if not topic:` here, it is more "Pythonic". :)